clients/include/testing_dense2csx.hpp

```diff
@@ -46,12 +46,11 @@ void testing_dense2csx_bad_arg(FUNC& dense2csx)
     // do not test for bad args
     return;
 #endif
-    static constexpr size_t               safe_size = 100;
-    static constexpr int                  M         = 10;
-    static constexpr int                  N         = 10;
-    static constexpr int                  LD        = M;
-    static constexpr hipsparseDirection_t dirA      = DIRA;
```

Comment:
Bug here where we used both the templated direction and this arguments direction